doc/Standaard-inrichting-GitHub-GitLab.md

```diff
@@ -1,6 +1,6 @@
 # Standaard inrichting GitHub/GitLab
 
-GitHub en GitLab repositories zijn langzamerhand de ruggegraat van ons werkproces. Voor bijna elk project is een GitHub en/of GitLab repository noodzakelijk.
+GitHub en GitLab repositories zijn langzamerhand de ruggewervel van ons werkproces. Voor bijna elk project is een GitHub en/of GitLab repository noodzakelijk.
```

Comment:
Het gezegde gebruikt de term ruggegraat, niet ruggewervel. Dat laatste is immers maar 1 klein onderdeeltje van de gehele graat. Ik stel dus voor dit niet te wijzigen.

Reply:
Touche, eigenlijk is de correcte term wervelkolom. Zoogdieren hebben nog steeds geen graten ;-)
Maar ik vind het prima het zo te laten.